src/main/features/response/form/models/defendantPaymentOption.ts

```diff
@@ -0,0 +1,62 @@
+import { IsDefined, IsIn } from 'class-validator'
+import { Serializable } from 'models/serializable'
+
+export class ValidationErrors {
+  static readonly OPTION_REQUIRED: string = 'Choose option'
```

Comment:
this looks like something that should be in the default `validationErrors`

Reply:
Definitely, what's more -- there already is one 😉